server/src/modules/auth/auth.routes.ts

```diff
@@ -65,5 +65,13 @@ export const routeAuth = ({
     controller.handleSendLoginOtp,
   )
 
+  /**
+   * Logout
+   * @route   POST /api/auth/logout
+   * @returns 200 if logged out
+   * @access  private
+   */
+  router.get('/logout', controller.handleLogout)
```

Comment:
don't think this should be a GET request, doesn't it modify the `sessions` table and hence change database state?

Reply:
Followed forms on this one. Happy to change to delete if you still think that is the way to go.

 https://github.com/opengovsg/FormSG/blob/232ff3ec94a090c3854e5b96a71c4d3f6dd036fe/src/app/routes/api/v3/auth/auth.routes.ts#L91

Reply:
You are right, `POST` should be used instead (not `DELETE` as there is no resource to identify). Similarly, the send and verify login should be `POST` but will not touch them here.